http/README.md

```diff
@@ -1,7 +1,7 @@
-# net
-
-Usage:
+# http
```

Comment:
You need to run format.ts

Reply:
@zhmushan thanks! Seems `format.ts` is not effecting `md` files inside module, so updated it too.